spring-batch-core/src/main/java/org/springframework/batch/core/repository/dao/JobInstanceDao.java

```diff
@@ -96,6 +96,16 @@
 	 */
 	List<JobInstance> getJobInstances(String jobName, int start, int count);
 
+	/**
+	 * Fetch the last job instance for the given job.
+	 * @param jobName name of the job
+	 * @return the last job instance if any or null otherwise
+	 */
```

Comment:
`@since` and define what "last" means

Reply:
It is the last job instance by id. I updated the Javadoc.